Set usage of TokenType.User as an error rather than a warning.

Joe4evr opened the pull request:
I think waiting for 3.0 to do this is taking too long. Discord disallowed external API access with user tokens since November, so we should be stopping users *completely* from continuing that practice ASAP.

Joe4evr committed:
Set usage of TokenType.User as an error rather than a warning.

AntiTcb commented:
They're going to be shut out of it in the future anyway, what's the point in delaying it when using the endpoints is a risk to their account's status?

Still34 commented:
Agreed. It is going to be shut off anyways.

Joe4evr committed:
Remove commented sections and #pragma's

Additionally, changes use of ReadMessages to ViewChannel since that Obsolete was also suppressed by the pragma
